lib/auth/webauthnwin/conv.go

```diff
@@ -50,7 +50,7 @@ func assertOptionsToCType(in wantypes.PublicKeyCredentialRequestOptions, loginOp
 		// https://github.com/microsoft/webauthn/blob/7ab979cc833bfab9a682ed51761309db57f56c8c/webauthn.h#L36-L97
 		// contains information about different versions.
 		// We can set newest version and it still works on older APIs.
-		dwVersion:                     6,
+		dwVersion:                     5,
```

Comment:
It seems like we only need to backport this to v18, right? I wasn't able to reproduce this in 15, 16, and 17.

Reply:
I think I'll backport it further, since while you couldn't reproduce it, it was possible it was just a luck; the version numbers are still inconsistent with structure sizes.